abdm_python_integrator/abha/tests/test_abha_verification.py

```diff
@@ -0,0 +1,130 @@
+from unittest.mock import Mock, patch
+
+from django.contrib.auth.models import User
+from django.test import override_settings
+from rest_framework import status
+from rest_framework.reverse import reverse
+from rest_framework.test import APITestCase
+
+
+class TestABHAVerification(APITestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.invalid_req_msg = "Unable to process the current request due to incorrect data entered."
+        cls.user = User.objects.create_superuser(username='test_user', password='test')
+
+    def setUp(self) -> None:
+        self.client.force_authenticate(user=self.user)
+
+    @staticmethod
+    def _mock_abdm_http_post(url, payload):
+        abdm_txn_id_mock = {"txnId": "1234"}
+        return {
+            "v1/search/searchByHealthId": {"authMethods": ["MOBILE_OTP"]},
```

Comment:
This should use constants as well

Reply:
Addressed in 277b9d17b5dfe5d0376cf986d8aee460b0322492